opencensus/plugins/grpc/internal/client_filter.cc

```diff
@@ -75,7 +72,8 @@ void CensusClientCallData::OnDoneRecvMessageCb(void *user_data,
   GPR_ASSERT(channeld != nullptr);
   // Stream messages are no longer valid after receiving trailing metadata.
   if ((*calld->recv_message_) != nullptr) {
-    calld->recv_message_count_++;
+    calld->context_.RecordMessageEvent(false, calld->recv_message_count_++,
```

Comment:
How about:
```
constexpr bool kRecv = false;
kSent = true;
```

Reply:
Can't inline RecordMessageEvent so that the call becomes `calld->context_.span_.AddReceivedMessageEvent(...)` ?

Reply:
Inlined now.

Reply:
Sorry, I meant inline the impl of CensusContext::RecordMessageEvent into this callsite, not the header file. :)